src/public-lib/lib/StringUtils.spec.ts

```diff
@@ -38,4 +38,16 @@ describe("unquote and decodeTabDelimited", () => {
         assert.equal(unquote(unescapeTabDelimited("\"cBioPortal\\n'Redefining'\\t\"Data Analysis\"\"")),
             "cBioPortal\n\'Redefining\'\t\"Data Analysis\"");
     });
+});
+
+
+describe("isUrl", () => {
+    it("gives correct result on various inputs", ()=>{
+        assert.isFalse(isUrl("example.com"))
```

Comment:
We don't want to support any link without protocol?

Reply:
that isn't supported. only http and https are supported
btw its `assert.isFalse`